src/test/java/TestNGTests/TestPost.java

```diff
@@ -0,0 +1,43 @@
+package TestNGTests;
+
+import java.io.IOException;
+import APICalls.PostCall;
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import utility.PropertyManager;
+
+public class TestPost extends PostCall {
```

Comment:
please check common comments that were added in assertJ and restAssured packages